src/renderer/tvgAnimation.cpp

```diff
@@ -62,6 +65,21 @@ Animation::Animation() : pImpl(new Impl)
 }
 
 
+Result Animation::override(const char* slotJson) noexcept
+{
+    if (!pImpl->overrider) {
+        auto loader = static_cast<LottieLoader*>(pImpl->picture->pImpl->loader);
```

Comment:
This should be something like this:
```
static_cast<LottieLoader*>(pImpl->picture->pImpl->loader)->override(slot);
```
Then, the LottieOverrider can reside within LottieLoader.